media/css/firefox/tiles.less

```diff
@@ -4,10 +4,136 @@
 
 @import "../sandstone/lib.less";
 
+#outer-wrapper {
+    overflow: hidden;
+}
+
 #wrapper {
     margin-top: 50px;
 }
 
-.sidebar .sidebar-inset img {
-    border: none;
+.sidebar {
+    float: left;
+    margin: -200px 10px 0 10px;
+
+    .sidebar-inset {
+        background-color: transparent;
+        box-shadow: none;
+        border: none;
+        padding: 0;
+
+        img {
+            border: none;
+            max-width: none;
+            margin-bottom: @gridGutterWidth;
+        }
+
+        img.tab {
+            width: 200%;
+        }
+
+        img.tab-preferences {
+            width: 100%;
+        }
+    }
+}
+
+.main-column {
+    width: 580px;
+
+    .intro {
+        font-size: @baseLine;
+        font-size: 1.2rem;
+        font-style: italic;
+        font-weight: 100;
+        padding-bottom: 2em;
+    }
+
+
+    section {
+        padding-top: 2.5em;
+        border-top: solid 3px @borderColor;
+
+        h4 {
+            font-weight: bold;
+            font-style: italic;
+        }
+
+        img {
+            margin: @gridGutterWidth 0;
+        }
+
+        &.tls-discover,
+        &.tls-started {
+            img {
+                margin-top: 0;
+            }
+        }
+        
+        ol {
+            list-style: none; 
+            counter-reset: array;
+            font-style: italic;
+            font-weight: bold;
+
+            li {
+                margin: 0px;
+            }
+
+            li:before {
+                counter-increment:array;
+                content: counter(array) ") ";
+            }
+
+            li:nth-child(2),
+            li:nth-child(3) {
+                float: left;
+                width: 46%;
+                margin-right: 20px;
+            }
+
+            li:nth-child(3) {
+                padding-left: .5em;
+                box-sizing: border-box;
+            }
+        }
```

Comment:
This is a tricky one but, I think we can clean this up a little so, couple of suggestions:

```
ol {
  font-family: "Open Sans";
  font-style: italic;
  font-weight: bold;
}
```
Let the `ol` just do the numbering and, seeing you are going to need `bold/italic` here again, it might very well be worth it to add the font to the page.

```
li:before {
    content: ") ";
}

With the `ol` doing the numbering, we only need to add the parenthesis. I am honestly not even sure if we should go that far, I know it is in the design but, I do not know whether it would be a problem to just have `1.` as apposed to '1.)'

I would also suggest using the images as a background image using the `:after' pseudo selector. Getting items two and three to line up is going to be a huge headache. You just need to set the container size so, that for items 2 and 3, you are cropping out the part of the image that is not relevant.

If you do that, you can then also get rid of the `li:nth-child(3)` rule.